src/index.css

```diff
@@ -23,9 +24,16 @@ li {
 
 header {
   display: flex;
-  justify-content: space-between;
-  padding: 10px 1rem;
+  padding: 10px 4rem;
   align-items: center;
+  gap: 2rem;
+  background-color: #fff;
+}
+
+header h3 {
+  font-size: 1.6rem;
+  cursor: pointer;
+  color: #0290ff;
```

Comment:
- [ ] The other issue is that we have to follow CSS best practices by writing DRY code. This means that we should use reusable CSS classes that can be reused in our components other than repeating styles. For example:
- The `background-color: #fff;` property is used three times.
- The `color: #121212;` property has been used six times
- The `display: flex;` property has been used eleven times

For more information kindly refer to this [resource](https://speckyboy.com/good-bad-css-practices/)

Reply:
- [ ] These suggested changes have not been implemented. The previous reviewer noticed that you are using same CSS rules many times and this is against DRY principle. what you can do is create classes like following and then just add this classname to any element you want to apply that specific styling and then remove these styling from other classnames.
example:
```
white {
  background-color: #fff;
},
flex {
  display: flex;
}
```
now you must place these classnames to elements you want to apply this styling and remove this specific repeated styling from other classes.

Reply:
- [ ] These suggested changes have not been implemented. The previous reviewer noticed that you are using same CSS rules many times and this is against DRY principle. what you can do is create classes like following and then just add this classname to any element you want to apply that specific styling and then remove these styling from other classnames.
example:
```
white {
  background-color: #fff;
},
flex {
  display: flex;
}
```
now you must place these classnames to elements you want to apply this styling and remove this specific repeated styling from other classes.